### PR TITLE
Allure report generation

### PR DESCRIPTION
**Pull request by netmajor**

### CODE REVIEW

**Comment by christian-bromann** [comment]
can you squash these commits into one?

**Comment by netmajor** [comment]
ok, i will create new branch with one commit

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "release:major": "np major",
     "test": "run-s eslint test:unit",
     "test:unit": "mocha",
+    "report": "node_modules/allure-commandline/bin/allure generate .allure-results --clean && node node_modules/allure-commandline/bin/allure open",
```
Comment (just-boris) [inline package.json]: Why not just `allure generate`?

Comment (netmajor) [reply package.json]: for me is important to see report, so I open it. Of course I could add opening report in new command, but faster is to do it at ones

Comment (christian-bromann) [reply package.json]: @netmajor I think what @just-boris is trying to say here is that you can leave out `node node_modules/allure-commandline/bin/` and just write `allure`. NPM should look into the right directory to find the executable

Comment (netmajor) [reply package.json]: When I don't specify .allure-result directory, it look into allure-directory which don't exist :/
By default report is not open by allure :/

Comment (christian-bromann) [reply package.json]: you can keep all the parameters , just remove `node_modules/allure-commandline/bin/`

Comment (netmajor) [reply package.json]: ok, i check this

Comment (netmajor) [reply package.json]: I need to remove current fork, because I FORGOT to rebase from master and have now mess :/ I back with better PR

Comment (netmajor) [reply package.json]: I get :
```
Error: Cannot find module '/Users/Netmajor/Documents/wdio-allure-reporter/allure'
```
So I need to leave this path

**Pull request closed without merging**